contributed_definitions/NXreflections.nxdl.xml

```diff
@@ -0,0 +1,415 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<!--
+  # NXreflections: candidate NeXus definition for reflection data
+-->
+<definition type="group" name="NXreflections" category="contributed" extends="NXobject"
+    xmlns="http://definition.nexusformat.org/nxdl/@NXDL_RELEASE@"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/@NXDL_RELEASE@ ../nxdl.xsd"
+    version="1.0"
+    >
+  <doc> This is a definition for reflection data from diffraction experiments </doc>
+  <group type="NXentry" minOccurs="1">
+    
+    <field name="definition">
+      <doc> NeXus NXDL schema to which this file conforms </doc>
+      <enumeration>
+        <item value="NXreflections" />
+      </enumeration>
+    </field>
+
+    <field name="experiments" type="NXmx" minOccurs="1">
+      <doc> The experiments from which the reflection data derives </doc>
+    </field>
+
+    <field name="h" type="NX_INT" minOccurs="1">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> The h component of the miller index </doc>
```

Comment:
In NXDL syntax, the ``<doc>`` element must come before any ``<attribute>`` elements.  This is the first instance. Apply this through remainder of this document.